lib/routes/51cto/recommend.ts

```diff
@@ -13,11 +18,46 @@ export const route: Route = {
         },
     ],
     name: '推荐',
-    maintainers: ['cnkmmk'],
+    maintainers: ['cnkmmk', 'ovo-tim'],
     handler,
     url: '51cto.com/',
 };
 
+const pattern = /'(WTKkN|bOYDu|wyeCN)':\s*(\d+)/g;
+
+async function get_fullcontent(item, cookie = '') {
+    let fullContent: null | string = null;
+    const articleResponse = await ofetch(item.url, {
+        headers: {
+            cookie,
+            'user-agent': config.ua,
+        },
+    });
+    const $ = load(articleResponse);
+    fullContent = item.url.includes('ost.51cto.com') ? $('.posts-content').html() : $('article').html();
+
+    if (!fullContent && cookie === '') {
+        // If fullContent is null and haven't tried to request with cookie, try to get fullContent with cookie
+        try {
+            // More details: https://github.com/DIYgod/RSSHub/pull/16583#discussion_r1738643033
+            const _matches = articleResponse!.match(pattern)!.slice(0, 3);
+            const matches = _matches.map((str) => Number(str.split(':')[1]));
+            const [v1, v2, v3] = matches;
+            const cookie = '__tst_status=' + (v1 + v2 + v3) + ';';
+            return await get_fullcontent(item, cookie);
+        } catch (error) {
+            logger.error(error);
+        }
+    }
+
+    return {
+        title: item.title,
+        link: item.url,
+        pubDate: parseDate(item.pubdate, +8),
+        description: fullContent || item.abstract + '(RSSHub: Failed to get fullContent)', // Return item.abstract if fullContent is null
```

Comment:
Do not add error messages to the article

Reply:
Fixed